README.md

```diff
@@ -20,16 +20,32 @@ import PropTypes from 'prop-types'; // ES6
 var PropTypes = require('prop-types'); // ES5 with npm
 ```
 
-If you prefer a `<script>` tag, you can get it from `window.PropTypes` global:
+### CDN
 
+If you prefer to exclude `prop-types` from your application and use it 
+globally via `window.PropTypes`, the `prop-types` package provides 
+single-file distributions, which are hosted on the following CDNs:
+
+* [**cdnjs**](https://cdnjs.com/libraries/prop-types)
+```html
+<!-- development version -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prop-types/15.5.10/prop-types.js"></script>
+
+<!-- production version -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/prop-types/15.5.10/prop-types.min.js"></script>
+```
+
+* [**unpkg**](https://unpkg.com/prop-types/)
 ```html
 <!-- development version -->
-<script src="https://unpkg.com/prop-types/prop-types.js"></script>
+<script src="https://unpkg.com/prop-types@15.5.10/prop-types.js"></script>
```

Comment:
Yes, valid point. It's the one downfall of using a specific revision.
React's documentation is out of sync with the latest... So are a bunch of other repositories that list this same "CDN" section in their Readme's.

If there's a more concise way to update the wording, I'm all for it. I pretty much just deferred to the React wording and am confident that anyone who is using a CDN will be smart enough to figure out how to automate that process into their build pipeline.